src/main/java/jenkins/advancedqueue/jobinclusion/strategy/FolderBasedJobInclusionStrategy.java

```diff
@@ -62,19 +66,70 @@ public ListBoxModel getListFolderItems() {
 
 	};
 
+	static public class JobPattern {
```

Comment:
Is it needed? I've been looking at how it's implemented in `ViewBasedJobInclusionStrategy` and it doesn't seem to be required there.